docs/util.md

```diff
@@ -483,7 +486,10 @@ In Java: Use `BigDecimal` to represent this value.
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| value | [double](#double) |  | Value in the range [0,1], inclusive |
+| rate | [string](#string) |  | Expects digits after a `.`, no + sign. Ex: 1.0, -1.0, 1, -1, 0.01. |
+| value | [double](#double) |  | **Deprecated.** deprecated
+
+Value in the range [0,1], inclusive |
```

Comment:
This formatted oddly: ![Screen Shot 2022-08-25 at 7 24 40 AM](https://user-images.githubusercontent.com/87503067/186691377-52ff638c-9110-4290-ae09-1e0751928264.png)

Reply:
The descriptions generated for deprecated fields are bugged and not outputting on the same line. I'll revert the changes generated from #15 and see about fixing all the instances in a single future PR.